src/Altinn.FileScan.Functions/Extentions/HttpClientExtension.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Altinn.FileScan.Functions.Extensions
+{
+    /// <summary>
+    /// This extension is created to make it easy to add a bearer token to a HttpRequests. 
+    /// </summary>
+    public static class HttpClientExtension
+    {
+        /// <summary>
+        /// Extension that add authorization header to request
+        /// </summary>
+        /// <param name="httpClient">The HttpClient</param>
+        /// <param name="requestUri">The request Uri</param>
+        /// <param name="content">The http content</param>
+        /// <param name="platformAccessToken">The platformAccess tokens</param>
+        /// <returns>A HttpResponseMessage</returns>
+        public static Task<HttpResponseMessage> PostAsync(this HttpClient httpClient, string requestUri, HttpContent content, string platformAccessToken = null)
```

Comment:
we will always require platform access token for this component, so maybe we can remove the default value